src/libmscore/scorefile.cpp

```diff
@@ -449,16 +449,7 @@ bool MasterScore::saveFile(bool generateBackup)
             dir.mkdir(backupSubdirString);
 #ifdef Q_OS_WIN
             const QString backupDirNativePath = QDir::toNativeSeparators(backupDirString);
-#if (defined (_MSCVER) || defined (_MSC_VER))
-   #if (defined (UNICODE))
-            SetFileAttributes((LPCTSTR)backupDirNativePath.unicode(), FILE_ATTRIBUTE_HIDDEN);
-   #else
-            // Use byte-based Windows function
-            SetFileAttributes((LPCTSTR)backupDirNativePath.toLocal8Bit(), FILE_ATTRIBUTE_HIDDEN);
-   #endif
-#else
-            SetFileAttributes((LPCTSTR)backupDirNativePath.toLocal8Bit(), FILE_ATTRIBUTE_HIDDEN);
-#endif
+            SetFileAttributes(reinterpret_cast<LPCTSTR>(backupDirNativePath.unicode()->unicode()), FILE_ATTRIBUTE_HIDDEN);
```

Comment:
that `->unicode()` is, at best, superfluos. If hasn't been need ever before, I don't see why it would now.

Reply:
https://github.com/musescore/MuseScore/pull/7744#discussion_r597465131

Reply:
> @igorkorsukov, `reinterpret_cast` was necessary to stop Qt Creator complaining.

`(LPCTSTR)` works just fine too.

Reply:
`(LPCTSTR) == reinterpret_cast` almost   :)
`(LPCTSTR)` - C style
`reinterpret_cast` - C++ stylr

Reply:
I got that (now)